homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -168,24 +183,102 @@ def lookup_name(mac: str) -> Optional[str]:
         # If an update is in progress, we don't do anything
         if update_bluetooth_lock.locked():
             _LOGGER.debug(
-                "Previous execution of update_bluetooth is taking longer than the scheduled update of interval %s",
+                "Previous execution of update_bluetooth "
+                "is taking longer than the scheduled update of interval %s",
                 interval,
             )
             return
 
         async with update_bluetooth_lock:
             await perform_bluetooth_update()
 
-    async def handle_manual_update_bluetooth(call):
-        """Update bluetooth devices on demand."""
-
-        await update_bluetooth()
-
     hass.async_create_task(update_bluetooth())
     async_track_time_interval(hass, update_bluetooth, interval)
 
-    hass.services.async_register(
-        DOMAIN, "bluetooth_tracker_update", handle_manual_update_bluetooth
```

Comment:
We can register this service in the component when that code is added. Then we use the dispatch helper to signal each, or a selected, Bluetooth adapter (each config_entry or a selected config_entry) to update.